### ISSUE TITLE
Tab reordering causes tab to disappear

### ISSUE THREAD

**Issue by cinnamon-msft** [microsoft/terminal#3661]
# Environment

```none
Windows build number: 10.0.18990.0
Windows Terminal version (if applicable): v0.6 on commit [71debc1](https://github.com/microsoft/terminal/commit/71debc158b6d90106a2dfc26b9029429499dfb29)

Any other software?
```

# Steps to reproduce
Drag the tabs around

# Expected behavior
I should be able to see the tab while dragging

# Actual behavior
No tab until I drop :(

![tab-reorder-bug](https://user-images.githubusercontent.com/48369326/69386670-208de380-0c78-11ea-9f2c-afa3894de3ef.gif)

**Comment by DHowett-MSFT** [comment]
Tracking MSFT:24126416

**Comment by Don-Vito** [comment]
@DHowett - the internal ticket is resolved for a year.. but the issue still persists.. 🤔

**Comment by DHowett** [comment]
@Don-Vito Unfortunately, it's only going to become available in GA when Windows builds newer than 19548 become GA. I cannot go on-record as to when that may be :grin:

**Comment by Don-Vito** [comment]
> 
> 
> @Don-Vito Unfortunately, it's only going to become available in GA when Windows builds newer than 19548 become GA. I cannot go on-record as to when that may be 😁

This is somewhat sad.. it took the dev a day to fix it.. a year ago.. and we still don't know when it will be released.. 🤦

**Comment by zadjii-msft** [comment]
This has been fixed for a while now. Looks like the build the fix went out in was 19548.1000.rs_prerelease.200114-1349, but I dunno how accurate that is.
